sample-xml/app/src/test/java/co/nimblehq/sample/xml/test/NavArgsExt.kt

```diff
@@ -0,0 +1,14 @@
+package co.nimblehq.sample.xml.test
+
+import androidx.navigation.NavArgs
+import co.nimblehq.sample.xml.extension.OverridableLazy
+import kotlin.reflect.KProperty1
+import kotlin.reflect.jvm.isAccessible
+
+fun <Arg : NavArgs, T> T.replace(
+    argumentDelegate: KProperty1<T, Arg>,
+    argument: Arg
+) {
+    argumentDelegate.isAccessible = true
+    (argumentDelegate.getDelegate(this) as OverridableLazy<Arg>).implementation = lazy { argument }
+}
```

Comment:
and this too

Reply:
Added in this https://github.com/nimblehq/android-templates/pull/450/commits/da634200a4bfe703fc2e947435c7f2ebf0c76c8d ✅